forge-gui/res/cardsfolder/a/archangel_of_strife.txt

```diff
@@ -8,8 +8,8 @@ SVar:ChooseEach:DB$ RepeatEach | RepeatPlayers$ Player | RepeatSubAbility$ DBCho
 SVar:DBChoice:DB$ GenericChoice | Defined$ Player.IsRemembered | Choices$ WarChoice,PeaceChoice
 SVar:WarChoice:DB$ Pump | Defined$ Player.IsRemembered | NoteCards$ Self | NoteCardsFor$ Strife | SpellDescription$ War
 SVar:PeaceChoice:DB$ Pump | Defined$ Player.IsRemembered | ClearNotedCardsFor$ Strife | SpellDescription$ Peace
-SVar:DBRemember:DB$ Pump | RememberObjects$ Player.NotedForStrife
+SVar:DBRemember:DB$ Pump | RememberObjects$ Player.NotedForStrife | ImprintCards$ Self
 S:Mode$ Continuous | AffectedZone$ Battlefield | Affected$ Creature.ControlledBy Remembered | AddPower$ 3 | Description$ Creatures controlled by players who chose war get +3/+0.
-S:Mode$ Continuous | AffectedZone$ Battlefield | Affected$ Creature.!ControlledBy Remembered | AddToughness$ 3 | Description$ Creatures controlled by players who chose peace get +0/+3.
+S:Mode$ Continuous | AffectedZone$ Battlefield | Affected$ Creature.!ControlledBy Remembered | AddToughness$ 3 | IsPresent$ Card.IsImprinted | Description$ Creatures controlled by players who chose peace get +0/+3.
```

Comment:
this doesn't work because it causes creatures that enter after to not apply

Reply:
I'm honestly only using it as a crutch to make sure it only applies if a choice was made on ETB, e.g. a copy with _Mirage Mirror_ should not buff anything
But yeah, a cleaner approach for the card in the future would be welcome

Reply:
we should probably extend so that it can be used for valid checks

it is probably something like Linked Ability Problem.

Also it probably shouldn't use `note` because the oracle text doesn't mention it

Reply:
I think the valids should be able to handle it already, hopefully just doing it with another Player field instead will work

but it's only used as helper during resolve currently, so maybe that can be improved together with #396 

though ideally it would somehow get all choices from the first linked ability